src/pages/Lyrics.js

```diff
@@ -1,25 +1,36 @@
 import { useState, useEffect } from "react";
+import "./Lyrics.css";
 
 export default function Lyrics() {
   const [lyrics, setLyrics] = useState(null);
   const [isLoading, setIsLoading] = useState(false);
+  const params = new URLSearchParams(document.location.search);
```

Comment:
I found this beautiful `custom hook` for this line of code in the  `react-router` [doc](https://reactrouter.com/web/example/query-parameters), so you don't have to use `document.location.search`)